src/drivers/uavcan/uavcan_main.cpp

```diff
@@ -901,9 +901,11 @@ int UavcanNode::run()
 
 			} else if (controls_updated && (_mixers != nullptr)) {
 
-				// XXX one output group has 8 outputs max,
+				// XXX one output group has 16 (esc_status_s::CONNECTED_ESC_MAX) outputs max (Q2/2019),
 				// but this driver could well serve multiple groups.
-				unsigned num_outputs_max = 8;
+				unsigned num_outputs_max = (
```

Comment:
Why is this necessary?

Reply:
You are right. I'm not sure about the need for this check.
Perhaps the best solution would be to assign the constant value 16 (esc_status_s::CONNECTED_ESC_MAX).
But, on the other hand, I did not want to bind to a constant, which in the distant future could still change.
If you think that the constant value would be better, I will be glad to change this.

Reply:
I think that the constexpr specifier solve all problems with CPU load and this check (CONNECTED_ESC_MAX constant presence) will be useful for further changes safety.